rds-monitoring/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for rds-monitoring.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+## The following settings are for the prometheus-cloudwatch-exporter chart
+prometheus-cloudwatch-exporter:
+  replicaCount: 1
+
+  serviceMonitor:
+    enabled: true
+    interval: 60s
+    labels:
+      prometheus: cloudwatch-exporter
+    timeout: 30s
+
+  resources:
+    limits:
+    memory: 320Mi
+  requests:
+    cpu: 10m
+    memory: 320Mi
+
+  aws:
+    region: eu-west-1
+    role:
+
+  serviceAccount:
+    create: true
+
+  rbac:
+    create: true
+
+  config: |-
+    region: eu-west-1
+    period_seconds: 60
```

Comment:
this interval might become costly -> to evaluate

Reply:
It's not the exporter's period_seconds that makes it costly, but the ServiceMonitor's scraping `interval` higher up.

Still valid comment though, as that's also set to 60 😅